### PR TITLE
feature: update table

### PR DESCRIPTION
**Pull request by Julhinha**
## Context

Jira Issue: 

ISEEC-751

Build Number: 

0.4.2

## Checklist:

- [ ] Verified in an integrated environment
- [ ] Unit and integration tests passing
- [ ] Semantic HTML with accessibility support
- [ ] No linting, test console, or browser console errors (best effort)
- [ ] JSDoc comment blocks for complex code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "react-dom": "16.13.1",
     "react-query": "^2.5.6",
     "react-router-dom": "5.1.2",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^4.0.0",
```
Comment (Okisa) [inline package.json]: Is there a reason for this version upgrade?
Version 4.0.0 is breaking tests, I think you should return to version 3.4.1 by now. Then you will be able to run `npm test` and fix any failing test.

File: src/Features/Overview/Policies/PoliciesTable/PoliciesTable.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ export default function PoliciesTable(props) {
       isSortable={true}
       render={({ rows, headers, getHeaderProps }) => (
         <TableContainer>
-          <Table className={styles.tableContainer} sortable={"true"} useZebraStyles={false}>
+          <Table className={styles.tableContainer} isSortable={"true"} useZebraStyles={false}>
```
Comment (Okisa) [inline src/Features/Overview/Policies/PoliciesTable/PoliciesTable.js]: You don't need to set `isSortable={"true"}`, just `isSortable` works.

File: src/Features/Overview/Violations/ViolationsTable/ViolationsTable.js [hunk 3]
```diff
@@ -201,7 +201,7 @@ export class ViolationsTable extends Component {
           isSortable={true}
           render={({ rows, headers, getHeaderProps, getRowProps }) => (
             <TableContainer>
-              <Table className={styles.tableContainer} sortable={"true"} useZebraStyles={false}>
+              <Table className={styles.tableContainer} isSortable={"true"} useZebraStyles={false}>
```
Comment (Okisa) [inline src/Features/Overview/Violations/ViolationsTable/ViolationsTable.js]: Same comment for `isSortable`.

File: src/Features/Templates/TemplatesTable/TemplatesTable.js [hunk 4]
```diff
@@ -77,7 +77,7 @@ export default function TemplatesTable(props) {
         isSortable={true}
         render={({ rows, headers, getHeaderProps }) => (
           <TableContainer>
-            <Table className={styles.tableContainer} sortable={"true"} useZebraStyles={false}>
+            <Table className={styles.tableContainer} isSortable={"true"} useZebraStyles={false}>
```
Comment (Okisa) [inline src/Features/Templates/TemplatesTable/TemplatesTable.js]: Same comment for `isSortable`.

**Review by Okisa — COMMENTED** [review]
Hey Julia! Nice changes here, just have a couple of comments, if you could take a look at them.

File: package.json (hunk 1 above)
Comment (timrbula) [reply package.json]: @isakawa-carbon thanks for the feedback on this. @Julhinha I do want to upgrade to 4.0 soon here (mainly for fast refresh support again) but that will likely be scheduled for the first thing after this release

**Review by timrbula — APPROVED** [review]
@Julhinha Please see the changes that I made based on the feedback from @isakawa-carbon and some things I encountered when reviewing the changes.

**Pull request merged**